UT4MasterServer/Models/GameServer.cs

```diff
@@ -70,119 +135,131 @@ public JObject ToJObject()
+	public bool AllowJoinInProgress { get; set; } = true;
 
+	[BsonElement("ShouldAdvertise")]
 	[JsonPropertyName("shouldAdvertise")]
-	public bool ShouldAdvertise { get; set; }
+	public bool ShouldAdvertise { get; set; } = true;
 
+	[BsonElement("IsDedicated")]
 	[JsonPropertyName("isDedicated")]
-	public bool IsDedicated { get; set; }
+	public bool IsDedicated { get; set; } = true;
 
+	[BsonElement("UsesStats")]
 	[JsonPropertyName("usesStats")]
-	public bool UsesStats { get; set; }
+	public bool UsesStats { get; set; } = false;
 
+	[BsonElement("UsesPresence")]
 	[JsonPropertyName("usesPresence")]
-	public bool UsesPresence { get; set; }
+	public bool UsesPresence { get; set; } = false;
 
+	[BsonElement("AllowInvites")]
 	[JsonPropertyName("allowInvites")]
-	public bool AllowInvites { get; set; }
+	public bool AllowInvites { get; set; } = true;
 
+	[BsonElement("AllowJoinViaPresence")]
 	[JsonPropertyName("allowJoinViaPresence")]
-	public bool AllowJoinViaPresence { get; set; }
+	public bool AllowJoinViaPresence { get; set; } = true;
 
+	[BsonElement("AllowJoinViaPresenceFriendsOnly")]
 	[JsonPropertyName("allowJoinViaPresenceFriendsOnly")]
-	public bool AllowJoinViaPresenceFriendsOnly { get; set; }
+	public bool AllowJoinViaPresenceFriendsOnly { get; set; } = false;
 
+	[BsonElement("BuildUniqueID")]
 	[JsonPropertyName("buildUniqueId")]
-	public string BuildUniqueID { get; set; }
+	public string BuildUniqueID { get; set; } = "256652735";
 
+	[BsonElement("LastUpdated")]
 	[JsonPropertyName("lastUpdated")]
-	public DateTime LastUpdated { get; set; }
+	public DateTime LastUpdated { get; set; } = DateTimeExtension.UnixTimestampStartOfTime;
 
+	/// <summary>
+	/// not sure what this is used for, perhaps we can remove it
+	/// entirely and just respond with static `"sortWeight": 0`
+	/// when needed
+	/// </summary>
+	[BsonElement("SortWeight")]
 	[JsonPropertyName("sortWeight")]
-	public int SortWeight { get; set; }
+	public int SortWeight { get; set; } = 0;
 
+	[BsonElement("Started")]
 	[JsonPropertyName("started")]
-	public bool Started { get; set; }
+	public bool Started { get; set; } = false;
 
 	public GameServer()
 	{
-		ID = EpicID.GenerateNew();
-		OwnerID = EpicID.GenerateNew();
-		OwnerName = "[DS]nohost-00000";
-		ServerName = "[DS]nohost-00000";
-		ServerAddress = "0.0.0.0";
-		ServerPort = 7777;
-		MaxPublicPlayers = 10000;
-		//OpenPublicPlayers = 10000;
-		MaxPrivatePlayers = 0;
-		//OpenPrivatePlayers = 0;
-		Attributes = new GameServerAttributes();
-		PublicPlayers = new List<EpicID>();
-		PrivatePlayers = new List<EpicID>();
-		//TotalPlayers = 0;
-		AllowJoinInProgress = true;
-		ShouldAdvertise = true;
-		IsDedicated = true;
-		UsesStats = false;
-		AllowInvites = true;
-		UsesPresence = false;
-		AllowJoinViaPresence = true;
-		AllowJoinViaPresenceFriendsOnly = false;
-		BuildUniqueID = "256652735";
-		LastUpdated = DateTimeExtension.UnixTimestampStartOfTime;
-		Started = false;
+		// everything is already initialized
```

Comment:
Remove empty constructor

Reply:
Because there is already another constructor the default one doesn't implicitly exist. Therefore we need to at least define it, else the other one with parameters is the only existing constructor.